api/tests/integration/application/users/user-controller_test.js

```diff
@@ -18,7 +18,6 @@ describe('Integration | Application | Users | user-controller', () => {
 
     sandbox.stub(usecases, 'getUserCampaignParticipationToCampaign');
     sandbox.stub(usecases, 'getUserProfileSharedForCampaign');
```

Comment:
juste pour info : `Since sinon@5.0.0, the sinon object is a default sandbox.` ([lien](https://sinonjs.org/releases/latest/sandbox/))
Je propose aucun changement ici mais ça va faire partie de mon nettoyage des tests je pense les sandbox 😄